docs/docs/rest-api/public/api/api.raml

```diff
@@ -0,0 +1,29 @@
+#%RAML 0.8
+
+title: Marathon REST API
+version: 2.0
+mediaType: application/json
+baseUri: /
+schemas:
+  - AppDefinition: !include v2/schema/AppDefinition.json
+  - Group: !include v2/schema/Group.json
+
+traits: !include traits.raml
+
+documentation:
+ - title: Using the Marathon REST API
+   content: Make me real!
```

Comment:
I need to click on the title in order to open a layover that renders the content. Feels like there should be text rendered directly underneath `Documentation` – expanding the `Using the Marathon REST API` makes no sense, and I don't think we need too much explanation for the documentation header.